erpnext/accounts/report/general_ledger/general_ledger.js

```diff
@@ -146,6 +146,12 @@ frappe.query_reports["General Ledger"] = {
 				return frappe.db.get_link_options('Project', txt);
 			}
 		},
+		{
+			"fieldname": "include_dims",
+			"label": __("Include Accounting Dimensions in Ledger"),
+			"fieldtype": "Check",
+			"default": 1
```

Comment:
```suggestion
			"default": 0
```
Should be unchecked by default